src/app/api/uploadthing/core.ts

```diff
@@ -0,0 +1,15 @@
+import { createUploadthing, type FileRouter } from "uploadthing/next";
+ 
+const f = createUploadthing();
```

Comment:
my favourite random variable name is `meep`. Just ask Aaron lol. Always best to name your variables with meaningful intention. The only caveat is the `for (let i = 0; i< blah blah)` those variables can be `i` or `j` or whatever :D